### PR TITLE
Fix:Inconsistency between the versioning in activity stream and docum…

### PR DESCRIPTION
**Pull request by mbenrhouma**
…ent application

### CODE REVIEW

File: core/webui/src/main/java/org/exoplatform/ecm/jcr/model/VersionNode.java [hunk 1]
```diff
@@ -114,7 +114,7 @@ private void addVersions(Node node, Session session) throws RepositoryException
         if(version.getUUID().equals(rootVersion.getUUID())) {
           continue;
         }
-        int versionIndex = Integer.parseInt(version.getName());
+        int versionIndex = Integer.parseInt(version.getName()) -1;
```
[the remark this answers is not in the document]
Comment (hghorbel) [reply core/webui/src/main/java/org/exoplatform/ecm/jcr/model/VersionNode.java]: The first version should be Version 1, then 2 then 3 and so on to keep the inconsistency with the version note displayed in the wiki's activity.

File: apps/portlet-explorer/src/main/webapp/groovy/webui/component/explorer/versions/UIVersionInfo.gtmpl [hunk 2]
```diff
@@ -92,7 +92,12 @@ import java.util.Locale;
       boolean isBaseVersion = uicomponent.isBaseVersion(node);
 
       String versionNumber = uicomponent.isRootVersion(node) ? uicomponent.getRootVersionNum() : node.getName();
-      String displayNumber = node.getDisplayName();
+      String displayNumber = "";
+        if (node.getDisplayName().equals("0")) {
+            displayNumber = _ctx.appRes("UIVersionInfo.tooltip.original");
+        }else {
+            displayNumber = node.getDisplayName();
+        }
```
Comment (thomasdelhomenie) [inline apps/portlet-explorer/src/main/webapp/groovy/webui/component/explorer/versions/UIVersionInfo.gtmpl]: Indentation is not good.

File: apps/portlet-explorer/src/main/webapp/groovy/webui/component/explorer/versions/UIVersionInfo.gtmpl [hunk 3]
```diff
@@ -92,7 +92,12 @@ import java.util.Locale;
       boolean isBaseVersion = uicomponent.isBaseVersion(node);
 
       String versionNumber = uicomponent.isRootVersion(node) ? uicomponent.getRootVersionNum() : node.getName();
-      String displayNumber = node.getDisplayName();
+      String displayNumber = "";
+        if (node.getDisplayName().equals("0")) {
```
Comment (thomasdelhomenie) [inline apps/portlet-explorer/src/main/webapp/groovy/webui/component/explorer/versions/UIVersionInfo.gtmpl]: Shouldn't we use "versionNumber" instead to check the version ?

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/versions/UIVersionInfo.java [hunk 4]
```diff
@@ -118,6 +118,7 @@ public void updateGrid() throws Exception {
       setRootVersionNum(String.valueOf(++lastVersionNum));
     } else {
       setRootVersionNum(String.valueOf(listVersion.size() + 1));
+      currentNodeTuple.setDisplayName("0");./
```
Comment (thomasdelhomenie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/versions/UIVersionInfo.java]: There are unexpected characters "./" at the end of the line.

File: apps/portlet-explorer/src/main/resources/locale/portlet/explorer/JCRExplorerPortlet_fr.xml [hunk 5]
```diff
@@ -1350,6 +1350,7 @@
       <empty>Cliquez ici pour ajouter un résumé</empty>
     </summary>
     <tooltip>
+      <original>Version Original</original>
```
Comment (thomasdelhomenie) [inline apps/portlet-explorer/src/main/resources/locale/portlet/explorer/JCRExplorerPortlet_fr.xml]: -> Version originale

File: apps/portlet-explorer/src/main/webapp/groovy/webui/component/explorer/versions/UIVersionInfo.gtmpl (hunk 3 above)
Comment (thomasdelhomenie) [reply apps/portlet-explorer/src/main/webapp/groovy/webui/component/explorer/versions/UIVersionInfo.gtmpl]: Comment not considered.

**Comment by mbenrhouma** [comment]
go to the versions list of the doc and restore v1 -> a version 4 is created but the version in the activity stream is 2  : 
This behaviour is not related to the fix, i will create another jira fir this issue.

**Comment by thomasdelhomenie** [comment]
ok, good.

**Pull request merged**